peachjam/models/generic_document.py

```diff
@@ -55,6 +55,16 @@ class Legislation(CoreDocument):
     class Meta:
         verbose_name_plural = "Legislation"
 
+    @property
+    def get_taxonomies(self):
+        return list(
+            self.taxonomies.distinct("topic").values_list("topic__name", flat=True)
+        )
+
+    @property
+    def get_year(self):
```

Comment:
A common distinction between properties and methods is that a method has a `get_...` but a property is just `year`. In this case, I think a property called `year` makes more sense.

Reply:
Done.